core/plugins/pacemaker.py

```diff
@@ -0,0 +1,73 @@
+import re
+
+from core import checks
+from core.plugintools import PluginPartBase
+from core.cli_helpers import CLIHelper
+
+PACEMAKER_PKGS_CORE = ['pacemaker', 'crmsh', 'corosync']
+PACEMAKER_SVC_EXPR = ['pacemaker[a-zA-Z-]*',
+                      'corosync']
+
+
+class PacemakerStatus(object):
+
+    @property
+    def crm_status(self):
+        return CLIHelper().pacemaker_crm_status
+
+    @property
+    def offline_nodes(self):
+        offline_nodes = []
+        crm_status = self.crm_status()
+        for line in crm_status:
+            regex_match = re.search(r'.*OFFLINE.*\[\s(.*)\s\]',
+                                    line)
+            if regex_match:
+                offline_nodes = regex_match.group(1).split()
+                return offline_nodes
+        return None
+
+    @property
+    def online_nodes(self):
+        online_nodes = []
+        crm_status = self.crm_status()
+        for line in crm_status:
+            regex_match = re.search(r'.*Online.*\[\s(.*)\s\]',
+                                    line)
+            if regex_match:
+                online_nodes = regex_match.group(1).split()
+                return online_nodes
+        return None
+
+
+class PacemakerBase(object):
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.status = PacemakerStatus()
+
+
+class PacemakerChecksBase(PacemakerBase, PluginPartBase):
```

Comment:
use this instead:

def __init__():
...
self.pacemaker = PacemakerBase()

Reply:
fixed on cab18285d94b5ececa730ae507fe39fc2d2da899